src/jobs/run.yml

```diff
@@ -36,6 +37,20 @@ parameters:
     enum: ["npm", "yarn", "yarn-berry"]
     default: "npm"
     description: Select the default node package manager to use. NPM v5+ Required.
+  start-command:
+    description: Command used to start your local dev server for Cypress to tests against
+    type: string
+    default: ""
+  cypress-command:
+    description: Command used to run your Cypress tests
+    type: string
+    default: "npx cypress run"
+  parallelism:
+    type: integer
+    default: 1
+    description: |
+      Number of Circle machines to use for load balancing, min 1
+      (requires "parallel" parameter set to true, and requires `record: true`)
```

Comment:
We technically want users to pass these flags moving forward right? So note about setting to true can be removed unless it's valuable in the interim.